changelog.md

```diff
@@ -33,6 +33,20 @@
 - react-native-bpk-component-icon: 1.1.15 => 1.2.0
   - `icons` object is now exposed, allowing users to refer to icon names using a constant.
 
+**Breaking:**
+- bpk-component-banner-alert:
+  - Removed `dismissable` property in favour of `configuration`.
+  - A banner alert with `children` will no longer be auotmatically expandable. Use the newly introduce `configuration` prop instead.
```

Comment:
> Removed `dismissable` property in favour of `configuration`.
> Use the newly introduce `configuration` prop instead.

These statements about the `configuration` prop don't make sense from a consumer POV. I.e. that prop isn't exposed to consumers. Instead it should focus on using one of the three new components (i.e. the contents of your migration guide).

Reply:
Yeah that's just me messing up